html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Eduroam.pm

```diff
@@ -0,0 +1,95 @@
+package pfappserver::Form::Config::Authentication::Source::Eduroam;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::Eduroam
+
+=cut
+
+=head1 DESCRIPTION
+
+Form definition to create or update an Eduroam authentication source.
+
+=cut
+
+use strict;
+use warnings;
+
+use HTML::FormHandler::Moose;
+
+extends 'pfappserver::Form::Config::Authentication::Source';
+with 'pfappserver::Base::Form::Role::Help';
+
+has_field 'tlrs1_server_address' => (
```

Comment:
You are right.
Here it is called TLRS by eduroam (as said, for Top-level RADIUS server) but it looks like it may also be called FLR (Federetion level RADIUS) elsewhere.
I'll have the name changed